index.html

```diff
@@ -5,41 +5,242 @@
+                        <span class="can3">CANOPY</span>
+                        <span class="img"><img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="back3">Back End Dev</span>
+                        <span class="img"> <img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="num3">2015</span>
+                    </div>
+                    <p class="to-do-para3">A daily selection of privately personalised reads: no accounts or sign-ups
+                        required.</p>
+                    <div class="works3">
+                        <div class="html3"> <a href="#">
+                                <p class="word413">html</p>
+                            </a></div>
+                        <div class="css3"><a href="#">
+                                <p class="word31">css</p>
+                            </a></div>
+                        <div class="jv3"> <a href="#">
+                                <p class="word3">javascript</p>
+                            </a></div>
+                    </div>
+                    <div class="click24">
+                        <button>See Projects</button>
+                    </div>
+                </section>
+
+                <!-- Part four -->
 
+                <section class="second-section51">
+                    <img src="./images/Snapshoot-Portfolio(6).jpg" alt="section-eader">
+                    <h1>Multi-Post Stories</h1>
+
+                    <div class="to-do51">
+                        <span class="can51">CANOPY</span>
+                        <span class="img"><img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="back51">Back End Dev</span>
+                        <span class="img"> <img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="num51">2015</span>
+                    </div>
+                    <p class="to-do-para51">A daily selection of privately personalised reads: no accounts or sign-ups
+                        required.</p>
+                    <div class="works51">
+                        <div class="html51"> <a href="#">
+                                <p class="word3151">html</p>
+                            </a></div>
+                        <div class="css51"><a href="#">
+                                <p class="word2151">css</p>
+                            </a></div>
+                        <div class="jv51"> <a href="#">
+                                <p class="word051">javascript</p>
+                            </a></div>
+                    </div>
+                    <div class="click51">
+                        <button>See Projects</button>
+                    </div>
+                </section>
+                
+                <!-- About myself -->
+                <section class="last">
+                    <h1>About <br> Myself</h1>
```

Comment:
Kindly change the heading of the about section to be any heading from `h2` to `h6`, for example: `h2` instead of `h1`. since the `h1` tag should only exist once on your page.